src/ts/utils.ts

```diff
@@ -36,7 +36,7 @@ export function formatQueryParams<T>(params?: Dict<T>): string {
     .filter(
       ([, value]: [string, T]) => value !== null && typeof value !== 'undefined'
     )
-    .map(([key, value]) => [key, value.toString()]);
```

Comment:
What was wrong with `toString`?

Reply:
Typescript didn't like it

Reply:
Then there's probably a bug.

Reply:
I think the bug was to call `toString` on an object as it's not 100% certain all objects will have that method. doing this will have the same result but will encapsulate all types.

Reply:
I guess the issue could be if this could produce unwanted behaviour, but if you look below this line, the value is concatinated with the key in the same way as I've changed it to here to be set in a query param, so I'm pretty sure this is the way to do it.

Reply:
What type is `value` inside of the filter function?

Reply:
Resolved in person.